tasks/Node/webpack.config.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+//  mode: "development",
```

Comment:
Do you want to clean up the commented out mode an devtool settings or did you want to leave them around?

Reply:
I figured we could leave it for the next person who may do some work and is just learning about Webpack. But I can definitely clean it up.

Reply:
Removed.